src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ActionPlanService.kt

```diff
@@ -82,6 +86,20 @@ class ActionPlanService(
       ?: throw EntityNotFoundException("action plan not found [referralId=$referralId]")
   }
 
+  private fun createEmptyAppointments(draftActionPlan: ActionPlan, submittedByUser: AuthUser) {
```

Comment:
nit: at the point this method is being called, the action plan is no longer a draft